packages/wrangler/src/dev/miniflare.ts

```diff
@@ -414,6 +414,10 @@ export function buildMiniflareBindingOptions(config: MiniflareBindingsConfig): {
 } {
 	const bindings = config.bindings;
 
+	// TODO(DEVX-1853, DEVX-1854): call `startMixedModeSession` here so that we get mixedModeConnectionStrings that we can
+	//       pass to miniflare (Note: this function is used from `LocalRuntimeController`, `MultiworkerRuntimeController` and
+	//       `getPlatformProxy`)
+
```

Comment:
I don't think we'd want to be calling it here, since this function is called on every reload. I think we'd probably want to call it directly in the controllers?